src/util/include/util/Util.h

```diff
@@ -85,6 +88,22 @@ constexpr const gchar* PRECISION_FORMAT_STRING = "%.8g";
 
 constexpr const auto DPI_NORMALIZATION_FACTOR = 72.0;
 
+/**
+ * Get the demangled name string of type T
+ */
+template <typename T>
+std::string demangledTypeName(T var) {
+    const char* mangledName = typeid(var).name();
+    char* demangledName = abi::__cxa_demangle(mangledName, nullptr, nullptr, nullptr);
```

Comment:
Which compilers are supported by Xournal++? It works on both GCC and Clang (on Linux), plus it also seemed to _build_ fine in the pipeline. I have seen somewhere that MSVC may not support it, but have not been able to verify it. Since MSVC's `std::type_info::name()` apparently returns a human-readable string, I can also just make it "passthrough" for MSVC.

Reply:
Thats not portable, cxxabi.h is part of the itanium api, which is not supported by msvc.